frontend/src/components/UserHome/UserHome.tsx

```diff
@@ -68,32 +77,39 @@ const UserHome: React.FC<{
           address(es). <Link route={{ type: "profile" }}>Check</Link>
         </Alert>
       )}
-
-      {data.primaryEmail ? (
-        <UserEmail email={data.primaryEmail} isPrimary />
-      ) : (
-        <Alert type="critical" title="No primary email adress" />
-      )}
-
-      {data.confirmedEmails.totalCount > 1 && (
-        <Body>
-          {data.confirmedEmails.totalCount - 1} additional emails.{" "}
-          <Link route={{ type: "profile" }}>View all</Link>
-        </Body>
-      )}
-
-      <Body>
-        {data.browserSessions.totalCount} active browser session(s).{" "}
-        <Link route={{ type: "browser-session-list" }}>View all</Link>
-      </Body>
-      <Body>
-        {data.oauth2Sessions.totalCount} active OAuth2 session(s).{" "}
-        <Link route={{ type: "oauth2-session-list" }}>View all</Link>
-      </Body>
-      <Body>
-        {data.compatSessions.totalCount} active compatibility layer session(s).{" "}
-        <Link route={{ type: "compat-session-list" }}>View all</Link>
-      </Body>
+      <section className="m-4 flex flex-col gap-2">
```

Comment:
Could you move some of those to classes in a CSS module? Overall I'm trying to reduce the amount of tailwind we use, especially for spacings and colors, and it feels like there is a lot of class repetition here